### PR TITLE
 Fix gRPC example with new syntax

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
Fix gRPC example with new syntax.

### CODE REVIEW

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out [hunk 1]
```diff
@@ -1,13 +1,14 @@
 # Create a Ballerina project and a module inside it.
-# Add grpc_bidirectional_streaming.bal file inside the module directory and
-# build the project.
-# For example, if you create a module named 'service', execute the following command to build
-# the 'service' module.
-$ ballerina build service
+# Generate mock service and stub code using grpc_bidirectional_streaming.proto file.
+# For example, if you create a module named 'service', execute the following command to generate ballerina code.
```
Comment (samgnaniah) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out]: Change "ballerina" to "Ballerina". Let's have this in title case always.

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+$ ballerina grpc --input grpc_bidirectional_streaming.proto --output service --mode service
+
+# Once you run the command, `grpc_bidirectional_streaming_pb.bal` and `Chat_sample_service.bal` files will be generated
```
Comment (samgnaniah) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out]: Let's say "... files are generated ...".  Avoid future tense wherever possible. Normally the word "will" is indicative of future tense. Apply elsewhere if applicable.

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_client.out [hunk 3]
```diff
@@ -1,11 +1,9 @@
...
-# Generate client stub code using .proto file generated from the service.
+# Generate client endpoint and stub code using grpc_bidirectional_streaming.proto file.
 # For example, if you create a module named 'client', execute the following command to generate stub code.
-$ ballerina grpc --input Chat.proto --output client --mode client
+$ ballerina grpc --input grpc_bidirectional_streaming.proto --output client --mode client
 
-# Once you run the command, the following files will be generated inside client directory.
-# * Chat_pb.bal
-# * Chat_sample_client.bal
+# Once you run the command, `grpc_bidirectional_streaming_pb.bal` and `Chat_sample_client.bal` files will be generated inside client directory.
```
Comment (samgnaniah) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_client.out]: Same comment as above.

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_pb.bal [hunk 4]
```diff
@@ -1,86 +1,65 @@
-// This is an auto generated client stub which is used to connect to gRPC server.
 import ballerina/grpc;
 import ballerina/io;
 
-// Non-blocking client.
-public type ChatStub object {
-    public grpc:Client clientEndpoint;
-    public grpc:Stub stub;
+public type ChatClient client object {
```
Comment (samgnaniah) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_pb.bal]: Shall we add more comments to this example?

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out [hunk 5]
```diff
@@ -1,13 +1,14 @@
 # Create a Ballerina project and a module inside it.
-# Add grpc_bidirectional_streaming.bal file inside the module directory and
-# build the project.
-# For example, if you create a module named 'service', execute the following command to build
-# the 'service' module.
-$ ballerina build service
+# Generate mock service and stub code using grpc_bidirectional_streaming.proto file.
```
Comment (samgnaniah) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out]: Let's add file names within single back ticks. Apply elsewhere if applicable.

File: examples/grpc-secured-unary/grpc_secured_unary.out [hunk 6]
```diff
@@ -1,13 +1,14 @@
 # Create a Ballerina project and a module inside it.
-# Add grpc_secured_unary.bal file inside the module directory and
-# build the project.
-# For example, if you create a module named 'service', execute the following command to build
-# the 'service' module.
-$ ballerina build service
+# Generate mock service and stub code using grpc_secured_unary.proto file.
```
Comment (samgnaniah) [inline examples/grpc-secured-unary/grpc_secured_unary.out]: Same comments as above.

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out (hunk 1 above)
Comment (daneshk) [reply examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out]: Fixed in 71f8f2d

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out (hunk 2 above)
Comment (daneshk) [reply examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out]: Fixed in 71f8f2d

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_client.out (hunk 3 above)
Comment (daneshk) [reply examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_client.out]: Fixed in 71f8f2d

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out (hunk 5 above)
Comment (daneshk) [reply examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out]: Fixed in 71f8f2d

File: examples/grpc-secured-unary/grpc_secured_unary.out (hunk 6 above)
Comment (daneshk) [reply examples/grpc-secured-unary/grpc_secured_unary.out]: Fixed in 71f8f2d

File: examples/grpc-secured-unary/grpc_secured_unary_pb.bal [hunk 7]
```diff
@@ -1,125 +1,89 @@
...
+    private grpc:Client grpcClient = new;
+    private grpc:ClientEndpointConfig config = {};
+    private string url;
 
-    public function init(grpc:ClientEndpointConfig config) {
-        // Initialize client endpoint.
+    function __init(string url, grpc:ClientEndpointConfig? config = ()) {
+        self.config = config ?: {};
+        self.url = url;
+        // initialize client endpoint.
```
[the remark this answers is not in the document]
Comment (daneshk) [reply examples/grpc-secured-unary/grpc_secured_unary_pb.bal]: Fixed in 71f8f2d

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_pb.bal (hunk 4 above)
Comment (daneshk) [reply examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming_pb.bal]: Fixed in 0980f96

File: examples/grpc-secured-unary/grpc_secured_unary_pb.bal [hunk 8]
```diff
@@ -1,125 +1,89 @@
-// This is an auto generated client stub, which is used to connect to the gRPC server.
 import ballerina/grpc;
 import ballerina/io;
 
-// Blocking client.
-public type HelloWorldBlockingStub object {
-    public grpc:Client clientEndpoint;
-    public grpc:Stub stub;
+public type HelloWorldBlockingClient client object {
```
[the remark this answers is not in the document]
Comment (daneshk) [reply examples/grpc-secured-unary/grpc_secured_unary_pb.bal]: Fixed in 0980f96

**Pull request merged**